src/main/java/edu/greenblitz/tobyDetermined/subsystems/Dashboard.java

```diff
@@ -36,26 +29,10 @@ public void periodic() {
 		SmartDashboard.putNumber("FL-angle-neo", Math.IEEEremainder(Math.toDegrees(SwerveChassis.getInstance().getModuleAngle(SwerveChassis.Module.FRONT_LEFT)), 360));
 		SmartDashboard.putNumber("BR-angle-neo", Math.IEEEremainder(Math.toDegrees(SwerveChassis.getInstance().getModuleAngle(SwerveChassis.Module.BACK_RIGHT)), 360));
 		SmartDashboard.putNumber("BL-angle-neo", Math.IEEEremainder(Math.toDegrees(SwerveChassis.getInstance().getModuleAngle(SwerveChassis.Module.BACK_LEFT)), 360));
-		SmartDashboard.putNumber("y", SwerveChassis.getInstance().getRobotPose().getY());
```

Comment:
Are you sure we want to delete all these logs from NT?

Reply:
yaaaas

Reply:
we already have the entire pose there